javatests/arcs/core/storage/BUILD

```diff
@@ -7,8 +7,11 @@ licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
 
+TEST_SRCS = glob(["*Test.kt"])
+
 arcs_kt_android_test_suite(
     name = "storage",
+    srcs = TEST_SRCS,
```

Comment:
This isn't strictly needed, it will glob all *.kt sources automatically

Reply:
In a previous iteration I had a fakes class that I ended up removing. This mirrors the structure we use in the handle tests and is a better glob pattern since it only picks up test classes.